pom.xml

```diff
@@ -1,16 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <parent>
-        <groupId>org.openhab</groupId>
-        <artifactId>pom</artifactId>
-        <version>2.5.0-SNAPSHOT</version>
-        <relativePath/>
-    </parent>
-
     <groupId>org.openhab.distro</groupId>
     <artifactId>pom</artifactId>
-    <!-- do not delete the version here as it is required by the release process -->
```

Comment:
Why did you remove this notice?

Reply:
Dropping the version of a POM makes only sense if you use a parent one.
If you don't use a parent POM, the version will not be removed at all.
So no need to state it explicit.

Or can you tell me the version of that coordinate if no parent is used and no one is set. :wink:

Reply:
Good point 🙄